test/test_autograd.py

```diff
@@ -8671,6 +8671,16 @@ def do_test():
 
         self.assertNotWarn(do_test)
 
+    def test_non_differentiable_ops(self, device):
```

Comment:
Note that we already have similar tests here: https://github.com/pytorch/pytorch/blob/7adb78017a81b656d22864ec97c072519a7dca63/test/test_autograd.py#L5507
We might want to have all of them together in the future.